packages/database/src/seed/data-dev.js

```diff
@@ -100,6 +100,14 @@ const lpaUsers = {
 		lpaCode: 'Q9999',
 		lpaStatus: 'added'
 	},
+	lpaUser2: {
+		id: '795ac593-4fe9-478c-8434-457db576a733',
+		email: 'user2@planninginspectorate.gov.uk',
+		isLpaUser: true,
+		isLpaAdmin: false,
+		lpaCode: 'Q1111',
+		lpaStatus: 'added'
```

Comment:
I havnt added this user in the mongo dev/data. Seemed to work just having the new test lpa in mongo data. Not sure if its needed as it seemed to work without (locally anyway)?

Reply:
Sounds alright, particularly if works